.gitmodules

```diff
@@ -105,3 +102,6 @@
 [submodule "projects/Intracellular_Recordings_of_Murine_Neocortical_Neurons"]
 	path = projects/Intracellular_Recordings_of_Murine_Neocortical_Neurons
 	url = https://github.com/conp-bot/conp-dataset-Intracellular-Recordings-of-Murine-Neocortical-Neurons.git
+[submodule "projects/Khanlab/BigBrainMRICoreg"]
```

Comment:
shouldn't this point to the conpdatasets organization instead of your username?

Reply:
Fair point, that was an error in reinstalling the reinitialised dataset.  Will fix.

Reply:
Have now updated to fix this.